src/App.jsx

```
@@ -1,14 +1,13 @@
-import React from 'react';
 import './App.scss';
 
+import { Sum } from './components/Sum/Sum';
+
 export const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum a={2} b={3} />
+    <Sum a={-5} b={5} />
+    <Sum a={10} />
+    <Sum b={5} />
+    <Sum />
```

Issue: The checklist requires both `a` and `b` props to be required for the `Sum` component. Here, you are rendering `<Sum />` with missing `a` or `b` props. Ensure that the `Sum` component throws an error when either prop is missing, as per the checklist: 'Both props must be required. If a prop is missing, the component should throw an error.' Please verify the implementation of `Sum` to enforce this behavior.